src/Diagnostics.RuntimeHost/Diagnostics.RuntimeHost.csproj

```diff
@@ -1,7 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
+    <UserSecretsId>79a3edd0-2092-40a2-a04d-dcb46d5ca9ed</UserSecretsId>
```

Comment:
Do we need this UserSecretsID?

Reply:
I left this in from trying to configure secrets.json, it won't hurt to leave it or remove it. Do you want me to take it out?